src/component/strategies/StrategiesList/StrategySwitch/StrategySwitch.tsx

```diff
@@ -0,0 +1,40 @@
+import { VFC } from 'react';
+import PermissionSwitch from 'component/common/PermissionSwitch/PermissionSwitch';
+import { UPDATE_STRATEGY } from 'component/providers/AccessProvider/permissions';
+import { Tooltip } from '@mui/material';
+
+interface IStrategySwitchProps {
+    deprecated: boolean;
+    onToggle: (state: boolean) => void;
+    disabled?: boolean;
+}
+
+export const StrategySwitch: VFC<IStrategySwitchProps> = ({
+    deprecated,
+    disabled,
+    onToggle,
+}) => {
+    const onClick = () => {
+        onToggle(deprecated);
+    };
+
+    const title = deprecated
+        ? 'Excluded from strategy list'
+        : 'Included in strategy list';
+
+    return (
+        <Tooltip
+            title={disabled ? 'You cannot disable default strategy' : title}
+            arrow
+        >
+            <span>
```

Comment:
Looks like `Tooltip` applies a label to the span, but not the input element. Maybe something like this would work:

https://github.com/Unleash/unleash-frontend/blob/main/src/component/feature/FeatureStrategy/FeatureStrategyIcon/FeatureStrategyIcon.tsx#L20-L24